go/vimlparser.go

```diff
@@ -3752,8 +3820,20 @@ func (self *Compiler) compile_subscript(node *VimNode) string {
 }
 
 func (self *Compiler) compile_slice(node *VimNode) string {
-	var r0 = func() string { if node.rlist[0] == nil {return "nil"} else {return self.compile(node.rlist[0]).(string)} }()
-	var r1 = func() string { if node.rlist[1] == nil {return "nil"} else {return self.compile(node.rlist[1]).(string)} }()
+	var r0 = func() string {
+		if node.rlist[0] == nil {
+			return "nil"
+		} else {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
if block ends with a return statement, so drop this else and outdent its block